src/test/java/org/mifos/integrationtest/cucumber/stepdef/BaseStepDef.java

```diff
@@ -5,11 +5,7 @@
 import java.security.PublicKey;
 import java.security.cert.X509Certificate;
 import org.mifos.connector.common.channel.dto.TransactionChannelRequestDTO;
-import org.mifos.integrationtest.config.BulkProcessorConfig;
-import org.mifos.integrationtest.config.ChannelConnectorConfig;
-import org.mifos.integrationtest.config.IdentityMapperConfig;
-import org.mifos.integrationtest.config.MockServer;
-import org.mifos.integrationtest.config.OperationsAppConfig;
+import org.mifos.integrationtest.config.*;
```

Comment:
Lets not do * imports.
[need to create a backlog issue to add code style in this repo]